public/index.html

```diff
@@ -175,8 +175,30 @@ <h4>Include links to privacy policy of Third Party Services <small>(Scroll for m
             </div>
           </section>
         </div>
+
         <div class="mdl-layout__tab-panel" id="privacypolicy">
           <section class="section--center mdl-grid">
+
+            <div class="mdl-cell mdl-cell--12-col mdl-shadow--2dp" id="fc-container">
+              <div id="fc-container-text">
+                <div style="color:#302428;">Free privacy policy hosting + website for {{appName}}.</div>
+                <div style="font-size:0.7em;color:#979193;">Powered by <a href="https://flycricket.com/?utm_source=partner&utm_medium=website&utm_campaign=privacy_generator" target="_blank">Flycricket</a></div>
+              </div>
+              <div id="fc-callouts">
+                <form action="https://app.flycricket.com/import/privacy" method="post" target="_blank" id="fc-form" onsubmit="return fc_submit(event)">
+                  <input type="hidden" name="fc-referrer" value="nishant">
+                  <input type="hidden" name="fc-body" id="fc-body">
+                  <input type="hidden" name="fc-name" id="fc-name" v-model="appName">
+                  <button class="mdl-button mdl-js-button mdl-button--raised" style="margin-right:8px;background:white;color:#00BCD4;">
+                    Learn More
+                  </button>
+                  <button class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent">
```

Comment:
There is no action for `Deploy` button?

Reply:
Actually both buttons do the same thing, because I don't have enough info to create an account and deploy directly from your page. The messaging and actions on the landing page works for both